quickstart.md

```diff
@@ -49,37 +45,55 @@ You now have a running CitrineOS server plus the supporting infrastructure, in t
+
+The `Boot` table has CRUD endpoints in order to be manipulated via REST API, of which the docs can be found here (if you have CitrineOS running locally): [localhost:8080/docs](http://localhost:8080/docs)
 Here is an example request for charge point `cp001`:
 
 ```shell
-curl --location --request PUT 'http://localhost:8081/data/provisioning/bootNotification?stationId=cp001' \
+curl --location --request PUT 'http://localhost:8080/data/configuration/boot?stationId=cp001' \
 --header 'Content-Type: application/json' \
---data '{"currentTime":"", "interval": 0, "status": "Accepted"}'
+--data '{ "status": "Pending" }'
 ```
 
-Now point your charger to your local machine and port `:8080` for the web-socket connection.
+#### Using Security Profile 1
+
+There are two layers of security available before the charger has the opportunity to send an OCPP message such as `BootNotificationRequest`. When the charger attempts to connect, it can be rejected at the transport layer if the websocket server is using security profiles 2 or 3. Otherwise, the http upgrade request occurs. The charger's upgrade request can be rejected if:
+
+- The request's subprotocol header is incorrect. The default websocket servers used by CitrineOS accept only 'ocpp2.0.1'.
+
+- The charging station's Id, as set in the url it connected with, is not known. This option can be toggled in the SystemConfig object used by CitrineOS, and is enabled by default only for the security profile 1 websocket server at `:8082`. To enter a charger into CitrineOS, use Directus. Navigate to the Charging Station collection, then create and save a new entry for your charging station. Make sure the Id of your new entry is the charging station's station id as set in the url it uses to connect to CitrineOS.
+
+- The websocket server is using security profiles 1 or 2 and the request's Authorization header has an incorrect username or password. The username will be checked against the charging station's id as set in the url it connected with. The password will be checked against the device model associated with the station id. Specifically, it will be the `Actual` Variable Attribute's value that belongs to the `BasicAuthPassword` Variable on the `SecurityCtrlr` Component. You can set this VariableAttribute on the CSMS side using the Variable Attribute CRUD endpoints on the Monitoring module. You can set this VariableAttribute on the Charging Station side using the SetVariables message, which can be sent from CitrineOS using the Monitoring module's message API.
```

Comment:
I found in the use case description, A01.FR.08, it mentions "the CSMS SHOULD only store salted password hashes, not the passwords themselves." Should we send password hash when set BasicAuthPassword when using CRUD endpoint?

Reply:
We do already! We can't take in a password hash in the PUT, because we'd have no way to un-hash it. But the GET and Directus only let you see the hashed password once you've set it.

Reply:
> We do already! We can't take in a password hash in the PUT, because we'd have no way to un-hash it. But the GET and Directus only let you see the hashed password once you've set it.

Got it. Thank you, Thana!